### DISCUSSION TITLE
Planning - Weekly Community Meeting

### DISCUSSION THREAD

**Discussion by shubham-cmyk** [kyverno/chainsaw#591] [General]
This is poll to decide the weekly meeting for chainsaw.

The normal poll won't work here since we have way too many options days/time so there would be too many combination. 
We could suggest our range of time in comments and the perfect time would be selected. 

`Note : Suggested Time should be in UTC`

Edit :- [The current Proposed time is Thursday 2PM UTC ](https://kubernetes.slack.com/archives/C067LUFL43U/p1702023635577239)

**Comment by shubham-cmyk** [comment]
I would suggest similar time of kyverno contibutor meeting `5pm UTC` on
 
Monday, Friday, Saturday or Sunday

**Reply by eddycharly** [reply]
`5pm UTC` is a bit late for me, let's see what works well for others.

Maybe we can share our timezone too ? Mine is `UTC+1`.

**Reply by shubham-cmyk** [reply]
Mine is UTC+5:30

**Reply by MariamFahmy98** [reply]
It can be 4pm or 3pm UTC on Thursday. WDYT?

**Reply by shubham-cmyk** [reply]
Works good for me.
Let's wait for more folks to comment

**Reply by anushkamittal2001** [reply]
Mine is UTC+530
3pm UTC Thursday works!

**Reply by shubham-cmyk** [reply]
@anushkamittal2001 It was 2PM UTC

**Reply by vishal-chdhry** [reply]
3PM or 4PM UTC on Thursday works for me
